plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8OpenShiftClientFactory.java

```diff
@@ -108,6 +115,12 @@ public Config buildConfig(Config defaultConfig, @Nullable String workspaceId)
             }
           }
         }
+      } else { // !runtimeIdentity.isPresent()
+        LOG.warn(
```

Comment:
Shouldn't we throw an exception here? If we can't get user identity OSO Proxy would not allow any operation, so we can just fail here.
@davidfestal WDYT?

Reply:
From discussion on mattermost, we're deferring this until later to avoid risking regressions.